x-pack/plugins/security_solution/cypress/e2e/detection_alerts/alert_table_action_column.cy.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { getNewRule } from '../../objects/rule';
+import {
+  CLOSE_OVERLAY,
+  OVERLAY_CONTAINER,
+  SESSION_VIEWER_BUTTON,
+  ANALYZER_VIEWER_BUTTON,
+} from '../../screens/alerts';
+import { createRule } from '../../tasks/api_calls/rules';
+import { cleanKibana } from '../../tasks/common';
+import { waitForAlertsToPopulate } from '../../tasks/create_new_rule';
+import { esArchiverLoad } from '../../tasks/es_archiver';
+import { login, visit } from '../../tasks/login';
+import { ALERTS_URL } from '../../urls/navigation';
+
+describe('Alerts Table Action column', { testIsolation: false }, () => {
+  before(() => {
+    cleanKibana();
+    esArchiverLoad('process_ancestry');
```

Comment:
After loading a new archive we need to unload them. Instead of adding a new document, is not possible to modify the existing one we load by default to have the needed fields? That would help us with the execution time.

Reply:
This was very careless of me. Thank you.

Additionally, yes I tried to modify existing data set but I had to create a separate because it was turning out to be lot of modifications on existing/default datasets.